### ISSUE TITLE
May not log in twice in same session in IE9

### ISSUE THREAD

**Issue by lloyd** [mozilla/persona#266]
Trying to log in twice to my favorite beer in IE9 does not work.  Steps to reproduce:
1. visit http://dev.myfavoritebeer.org/
2. click sign in
3. log in using an email address via browserid
4. click logout
5. click sign in again

Notice that the second time the dialog comes up after step 5  it has no content in the white area and is unusable.

reproduced on win7/IE9

**Comment by shane-tomlinson** [comment]
Duplicate of #243, which was fixed yesterday.  Closing
